README.md

```diff
@@ -100,7 +107,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a table created to manage data attributes, assign unique id, link other data and set restrictions on data values/input.
```

Comment:
This is close, but a schema defines the tables in a database, not just one table. And a schema isn't a table itself, it's just a description of the tables.